site/content/docs/4.3/examples/dashboard/dashboard.css

```diff
@@ -22,6 +22,13 @@ body {
   box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
 }
 
+@media (max-width: 767px) {
```

Comment:
This should probably be `max-width: 767.98px` so that we are consistent with core